client/src/components/AddItem.js

```diff
@@ -54,9 +55,40 @@ const useStyles = makeStyles((theme) => ({
 
 const demoListsArray = ["Clothes", "Furniture", "Luxury"];
 
+const getItem = async (input) => {
+  const response = await fetch("/scrape", {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({ URL: input }),
+  });
+  return response.json();
+};
+
+const emptyItem = {
+  shopURL: null,
+  title: null,
+  oldPrice: null,
+  price: null,
+  imgURL: null,
+};
+
 const AddItem = () => {
-  const [selectedItem, setSelectedItem] = useState("none");
+  const [inputLink, setInputLink] = useState("");
+  const [item, setItem] = useState(emptyItem);
+  const [popupOpen, setPopupOpen] = useState(false);
+  const [selectedList, setSelectedList] = useState("none");
   const classes = useStyles();
+  const openPopup = () => setPopupOpen(true);
+  const addButtonClick = async () => {
+    // regex check inputLink here
```

Comment:
The common practice is to preface comments like this with "TODO:". Also, we may want to also add in the future to check that the link has a domain that we are able to scrape